docs/Using-Fleet/MDM-macOS-setup.md

```diff
@@ -1,5 +1,7 @@
 # macOS setup
 
+_Available in Fleet Premium_
+
 In Fleet, you can customize the first-time macOS setup experience for your end users:
```

Comment:
The phase "first-time" here is a little unclear and if readers don't understand it might cause them to think that ABM is required even if they don't install fleet during the out of box setup. A different phase could make your changes easier to understand.

Reply:
Yeah, I see what you're saying. I like "out-of-the-box." I updated the docs to use this instead.

Thanks for the feedback!